### ISSUE TITLE
Colorblind mode should be disabled by default

### ISSUE THREAD

**Issue by xDoge26** [mastercoms/tf2cfg#773]
**Describe the bug**
Hello i dont know why would mastercomfig is forcing colorblind enabled. I would like a toggle for it

**How you encountered the issue**
Using mastercomfig low

**Screenshots**

**Comment by mastercoms** [comment]
colorblind mode is simply just an indicator over a player when they are under the effect of jarate or mad milk, and it's simple enough to override the console variable in your own autoexec.

**Comment by xDoge26** [comment]
Yeah i know that but it doesnt explain why it's enabled by default in low preset with and require config file to disable.

**Comment by mastercoms** [comment]
Well normally I have been against modules which only contain one console variable, in favor of making a system beyond modules that is able to control settings like that. Also, I haven't really prioritized it given that the Jarate/Mad Milk indicator can be considered an advantage / helpful feature and doesn't affect performance from what I've seen, and I have not heard people requesting it be disabled much over the years.

**Comment by xDoge26** [comment]
It could give player visibility disadvantage in a community server, which usually has an abnormally high player count.

**Comment by treacherousfiend** [comment]
I could be wrong, but as far as I'm aware we prioritize the experience on vanilla servers above that of community servers, where a whole host of things could be different.
For example we disable the flashlight by default since its technically just a holdover from Half-Life 2, and only custom gamemodes like Slender Fortress use it. This arguably isn't the best example though, since it does have a module now, but didn't used to, and because it has a tangible performance benefit to keep disabled for the people not playing on servers that use it.
As coms said though, its just one console variable, and the people who don't want it are more than encouraged to disable it in their autoexec files, but as it stands I think the large _large_ majority of players don't even touch community servers, let alone ones with 32+ players.
In my personal opinion, potential advantage for 95% of players outweighs potential disadvantage for 5% in this case.
